server/data_access_layer/models/localdb/localdb.model.js

```diff
@@ -189,21 +197,69 @@ module.exports = (localdb, Sequelize) => {
         }
     });
 
-    ChartReport.belongsTo(User, {
+    ChartReportData.belongsTo(ChartReport, {
         foreignKey: {
-            name: 'user_user_id',
-            allowNull: false,
+            name: 'chart_report_id',
+            allowNull: false
         },
         onDelete: 'CASCADE'
     });
 
-    ChartReportData.belongsTo(ChartReport, {
+    const ReportType = localdb.define("report_types", {
+        reportTypeId: {
+            field: 'report_type_id',
+            type: Sequelize.UUID,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true
+        },
+        reportTypeName: {
+            field: 'report_type_name',
+            type: Sequelize.STRING
+        },
+        frequency: {
+            field: 'frequency',
+            type: Sequelize.INTEGER,
+            defaultValue: 0
+        }
+    });
+
+    const Recipients = localdb.define("recipients", {
+        recipientId: {
+            field: 'recipient_id',
+            type: Sequelize.UUID,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true
+        },
+        name: {
+            field: 'recipient_name',
+            type: Sequelize.STRING
+        },
+        email: {
+            field: 'recipient_email',
+            type: Sequelize.STRING
+        }
+    })
+
+    const ReportTypeRecipients = localdb.define("report_type_recipients", {
```

Comment:
You can put the constant whole statement in one line if there is no key value pair declared inside,